scripts/get-version-matrix.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+function get_latest_version() { 
+    curl -s https://api.github.com/repos/hashicorp/terraform/git/refs/tags | \
```

Comment:
I just thought of something after hitting "approve" -- sorry about the timing. But I noticed there's a releases API too. https://docs.github.com/en/rest/reference/repos#list-releases

Reply:
Should we maybe use releases instead of tags, just in case the tagged version is not yet in the registry? They have 192 tags and 192 releases. That probably means you're safe enough to keep it as-is. But I thought I'd mention it anyway.

Reply:
This is a good point, although using either releases or tags is going to have the same the problem that if the binary is not yet uploaded to the releases CDN then the tests will fail because tfinstall wont be able to fetch that version yet.

Something we could do is actually hit [releases.hashicorp.com/terraform](https://releases.hashicorp.com/terraform) for to get the version numbers. The problem there is that the output is HTML so we'll have to scrape it the ye old fashioned way. I know ENGSRV (see ENGSRV-055) are working on an API for release metadata but it's not yet out of the oven.